Recipes/Recipes.API/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "Default": "Host=localhost;Port=5432;Database=RecipesDB;Username=postgres;Password=postgres;"
```

Comment:
Maybe we could leave this generic and move the secrets to the secrets file

Reply:
I think it's okay for development purposes